index.html

```diff
@@ -0,0 +1,88 @@
+    <title>Alef's Portfolio</title>
+</head>
+<body>
+
+    <header>
+        <a href="">
+            <img src="./assets/svg/Logo-Placeholder.svg" alt="site logo">
+        </a>
+        <a href="#" class="header--menu--icon">
+            <img src="/assets/svg/Icon-Menu.svg" alt="menu icon">
+        </a>
+    </header>
+    <main>
+        <section class="headline--container">
+            <div class="headline--title">
+                <h2>Aléf Garrido</h2>
+                <h1>Software Developer</h1>
+                <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+            </div>
+            <span class="devider">
+                <img src="/assets/svg/Devider.svg" alt="Section devider">
+            </span>
+            <section class="skills--container">
+                <div class="skills--item">
+                    <h3>Language</h3>
+                    <p>Javascript</p>
+                    <p>Ruby</p>
+                    <p>Html</p>
+                    <p>Css</p>
+                </div>
+                <div class="skills--item">
+                    <h3>Frameworks</h3>
+                    <p>Bootstrap</p>
+                    <p>Ruby on Rails</p>
+                    <p>RSPec</p>
+                    <p>CapyBara</p>
+                    <p>Selenium</p>
+                </div>
+                <div class="skills--item">
+                    <h3>Skills</h3>
+                    <p>Codekit</p>
+                    <p>Github</p>
+                    <p>Codepen</p>
+                    <p>Gitlab</p>
+                    <p>Terminal</p>
+                </div>    
+            </section>
+            
+            <span class="devider">
+                <img src="/assets/svg/Devider.svg" alt="Section devider">
+            </span>
+        <div class="supporting-item-container">    
+            <div class="social--icons--container">
+                <a href="#">
+                    <img src="/assets/svg/Icon-GitHub.svg" alt="Github Link">
+                </a>
+                <a href="#">
+                    <img src="/assets/svg/Icon-Linkedin.svg" alt="LinkedIn Link">
+                </a>
+                <a href="#">
+                    <img src="/assets/svg/Icon-Angelist.svg" alt="AngelList Link">
+                </a>
+                <a href="#">
+                    <img src="/assets/svg/Icon-Twitter.svg" alt="Twitter Link">
+                </a>
+                <a href="#">
+                    <img src="/assets/svg/Icon-Medium.svg" alt="Medium Link">
+                </a>  
+            </div>
+            <a class="supporting-item"  href="#">
+                <img src="/assets/svg/Element-Right.svg" alt="section link">
+            </a>
+        </div>    
```

Comment:
I suggest you wrap this section using the ` ul`  tag to group the social icons, and `li` per icon; each icon has a link (a tag)